### PR TITLE
Studio Desktop v2.1.0 announcement

### PR DESCRIPTION
**Pull request by philsturgeon**

### CODE REVIEW

**Comment by lottamus** [comment]
Deploy preview for *stoplightio* ready!

Built with commit bd8cb8053e540e3de57f4cbf587ce17f73a161c1

https://deploy-preview-1642--stoplightio.netlify.app

File: netlify/blog-posts/studio-v2-1-0.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  twitter:
+    description: > 
+      Stoplight Studio v2.1.0 has a few little features you'll love: custom table of contents, creating OpenAPI/JSON Schema from pasted in JSON, and a myriad of performance improvements.
+    title: Stoplight Studio Desktop v2.1 | Stoplight API Intersection
+    image: /images/blog/studio-and-platform-v2-0/cover.png
+    username: '@stoplightio'
+---
+
+
+We've just released a new version of Stoplight [Studio Desktop](/studio), our standalone application for Windows/macOS/Linux which allows you to manage OpenAPI, JSON Schema, AsyncAPI, and Markdown documents locally. [Studio Desktop v2.1.0](https://github.com/stoplightio/studio/releases) comes with a few new features we'd like to show off.
```
Comment (texas-kristen) [inline netlify/blog-posts/studio-v2-1-0.md:30]:
```suggestion
We've just released a new version of Stoplight [Studio Desktop](/studio), our standalone application for Windows/macOS/Linux, which allows you to manage OpenAPI, JSON Schema, AsyncAPI, and Markdown documents locally. [Studio Desktop v2.1.0](https://github.com/stoplightio/studio/releases) comes with a few new features we'd like to show off.
```

File: netlify/blog-posts/studio-v2-1-0.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+## Generate Schema from JSON
+
+Have you ever looked at a API response or some other chunk of JSON and wished you could automagically create an OpenAPI / JSON Schema model from it? Now you can!
```
Comment (texas-kristen) [inline netlify/blog-posts/studio-v2-1-0.md:34]:
```suggestion
Have you ever looked at an API response or some other chunk of JSON and wished you could automagically create an OpenAPI / JSON Schema model from it? Now you can! 
```

File: netlify/blog-posts/studio-v2-1-0.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+## Generate Schema from JSON
+
+Have you ever looked at a API response or some other chunk of JSON and wished you could automagically create an OpenAPI / JSON Schema model from it? Now you can! 
+
+![Inside Studio when editing the body a new "Generate from JSON" button is available.](/images/blog/studio-v2.1/generate-model.png)
+
+Stoplight Studio will take a look at each of the properties in that JSON, see what data is there, and make the most accurate model it can. It mostly looks out for properties, basic data types, and lets you decide what other validations can be applied later. This feature will hopefully save your keyboard from a bunch of distress.
+
+Why would you need to do this? Maybe you made the API already and now you're trying to document it. Or you're one of the thousands of API developers who used to be [entirely code-first](https://stoplight.io/blog/generate-documentation-from-code/), but are now migrating from a [code-first workflow to a design-first workflow](https://stoplight.io/blog/api-design-first-vs-code-first/).
```
Comment (texas-kristen) [inline netlify/blog-posts/studio-v2-1-0.md:40]:
```suggestion
Why would you need to do this? Maybe you made the API already, and now you're trying to document it. Or you're one of the thousands of API developers who used to be [entirely code-first](https://stoplight.io/blog/generate-documentation-from-code/) but are now migrating from a [code-first workflow to a design-first workflow](https://stoplight.io/blog/api-design-first-vs-code-first/). 
```

File: netlify/blog-posts/studio-v2-1-0.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+Documentation hosted on Stoplight has a default table of contents created for it, but if you want **more power**, you can replace that with an entirely custom one! 
+
+![The format of the table of contents is a toc.json file with an "items". More information can be found in the documentation linked below.](/images/blog/studio-v2.1/studio-copy-path.gif)
+
+Learn more about [custom Table of Contents](https://meta.stoplight.io/docs/platform/4.-documentation/d.table-of-contents.md) in our documentation.
+
+## Performance Improvements
+
+Our engineers have been hard at work, benchmarking and debugging various parts of our codebase, and making code contributions to the various open-source tools we rely upon, all with the goal of making Studio perform faster, and use less memory. There's too many tweaks to get into here, but things should feel smoother whilst you're editing large API descriptions.
```
Comment (texas-kristen) [inline netlify/blog-posts/studio-v2-1-0.md:52]:
```suggestion
Our engineers have been hard at work, benchmarking and debugging various parts of our codebase and making code contributions to the various open-source tools we rely upon, all with the goal of making Studio perform faster and use less memory. There are too many tweaks to get into here, but things should feel smoother while you're editing large API descriptions. 
```

File: netlify/blog-posts/studio-v2-1-0.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+Learn more about [custom Table of Contents](https://meta.stoplight.io/docs/platform/4.-documentation/d.table-of-contents.md) in our documentation.
+
+## Performance Improvements
+
+Our engineers have been hard at work, benchmarking and debugging various parts of our codebase, and making code contributions to the various open-source tools we rely upon, all with the goal of making Studio perform faster, and use less memory. There's too many tweaks to get into here, but things should feel smoother whilst you're editing large API descriptions. 
+
+## Non-JSON Examples
+
+JSON is easily the most commonly used data format for Web APIs, but sometimes plain-text, CSV, and other text-based formats are used too. In Studio v2.1.0 we make it possible for any response mime type to contain examples.
```
Comment (texas-kristen) [inline netlify/blog-posts/studio-v2-1-0.md:56]:
```suggestion
JSON is easily the most commonly used data format for Web APIs, but sometimes plain-text, CSV, and other text-based formats are used too. In Studio v2.1.0, we make it possible for any response mime type to contain examples.
```

File: netlify/blog-posts/studio-v2-1-0.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+
+![](/images/blog/studio-v2.1/non-json-examples.png)
+
+This is the first of a few changes to expand our tooling beyond a focus on JSON. Soon we'll be rolling out some impressive changes to documentation and the preview panel, which will make things work better, look better, for APIs that do not use JSON, or have a mixture of mime types.
```
Comment (texas-kristen) [inline netlify/blog-posts/studio-v2-1-0.md:60]:
```suggestion
This is the first of a few changes to expand our tooling beyond a focus on JSON. We’ll soon be rolling out some impressive changes to documentation and the preview panel, which will make things work and look better for APIs that do not use JSON or have a mixture of mime types.
```

File: netlify/blog-posts/studio-v2-1-0.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+
+---
+
+To download the latest version of Studio Desktop, visit the download page [here](https://github.com/stoplightio/studio/releases). 
+
+Want to keep an eye on what features we're considering, what's currently being developed, and what's been recently released? Check out our [public roadmap](https://roadmap.stoplight.io) where you can submit your ideas, and vote on what we should build next!
```
Comment (texas-kristen) [inline netlify/blog-posts/studio-v2-1-0.md:66]:
```suggestion
Curious about what features we’re considering, currently developing, and recent releases? Check out our [public roadmap](https://roadmap.stoplight.io) to submit your ideas, and vote on what we should build next!
```

**Review by texas-kristen — APPROVED** [review]

**Pull request merged**